terraform/dotnet/ec2/windows/main.tf

```diff
@@ -0,0 +1,221 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+  }
+}
+
+provider "aws" {}
+
+resource "aws_default_vpc" "default" {}
+
+resource "tls_private_key" "ssh_key" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+resource "aws_key_pair" "aws_ssh_key" {
+  key_name   = "instance_key-${var.test_id}"
+  public_key = tls_private_key.ssh_key.public_key_openssh
+}
+
+data "aws_ami" "ami" {
+  owners      = ["amazon"]
+  most_recent = true
+  filter {
+    name   = "name"
+    values = ["Windows_Server-2019-English-Full-Base-*"]
```

Comment:
Have we made sure this won't change over time? Is `2019` expected?

Reply:
2019 and 2022 are both supported version now for dotnet and for AWS, that just a place holder. If AWS decided to deprecate 2019 in the future, will do that. 
BTW let me know if you think should put 2022. No different

Reply:
Should ask @vastin that question

Reply:
confirmed, prefered 2022, changed